src/lib.rs

```diff
@@ -19,15 +19,15 @@
 //! use gfx_glyph::{Section, Layout, GlyphBrushBuilder};
 //! # fn main() {
 //! # let events_loop = glutin::EventsLoop::new();
-//! # let (window, mut device, mut gfx_factory, mut gfx_target, mut main_depth) =
+//! # let (_window, _device, mut gfx_factory, gfx_target, _main_depth) =
 //! #     gfx_window_glutin::init::<gfx::format::Srgba8, gfx::format::Depth>(
 //! #         glutin::WindowBuilder::new(),
 //! #         glutin::ContextBuilder::new(),
 //! #         &events_loop);
 //! # let mut gfx_encoder: gfx::Encoder<_, _> = gfx_factory.create_command_buffer().into();
 //!
-//! let arial = include_bytes!("examples/Arial Unicode.ttf");
-//! let mut glyph_brush = GlyphBrushBuilder::using_font(arial)
+//! let arial = include_bytes!(concat!(env!("CARGO_MANIFEST_DIR"), "/examples/Arial Unicode.ttf")).as_ref();
```

Comment:
I assume you're using nightly? Relativity has changed for the rust doc compiler in version 1.21. However since this only affects running the tests with nightly, rather than compiling the code I prefer to leave it in the cleanest form compatible with stable. 

When 1.21 is stable it'll be simple to change to the new/better relative path, and still as clean.